ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/tls/client/PicoCliClientTlsOptions.java

```diff
@@ -64,4 +69,49 @@ public boolean isCaAuthEnabled() {
   public Optional<Path> getKnownServersFile() {
     return Optional.ofNullable(knownServersFile);
   }
+
+  public boolean isTlsEnabled() {
+    return tlsEnabled;
+  }
+
+  public String validationMessage() {
```

Comment:
nit: feel like the validation should reside _on_ the structure - maybe in a separate class? Thus, this is just a data struct, with no concepts of valid/invalid